docs/knowledgebase/runtime/macros.md

```diff
@@ -95,9 +95,11 @@ To define storage items in a FRAME pallet. A storage item definition includes:
   - for `StorageMap`: `map hasher($hasher) rust_type => rust_type`
   - for `StorageDoubleMap`:
     `doublemap hasher($hasher) rust_type, hasher($hasher) rust_type => rust_type`
+  - for `StorageNMap`:
+    `nmap hasher($hasher1) rust_type, hasher($hasher2) rust_type, /* ... */ hasher($hasherN) rust_type => rust_type`
```

Comment:
I don't actually have a good syntax to indicate that the `hasher($hasher) rust_type` part is repeatable here, not sure what I have on this line correctly captures the intent, or whether I needed to write more to explain that this part is repeatable.